bin/init-eg-stack

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -u
+
+nodes=(db-coord-1 db-coord-2 db-data-1 db-data-2)
+#-------------------------------------------------------------------------------
+function log() {
+    local msg="$*"
+
+    echo "$(date -u '+%Y-%m-%dT%H:%M:%S%z'), $msg"
+}
+
+function startup() {
+    log "========================================"
+}
+
+function shutdown() {
+    log "----------------------------------------"
+}
+
+function srv() {
+    local node=$1
+    shift
+
+    kubectl exec "$node" -i -t -- "$@"
```

Comment:
I guess it's already worth noting that this is not Kubernetes per se, but also using Kubectl, which I guess is pretty standard, but I don't *think* (perhaps someone else knows more than me about this?) that it's actually strictly necessary for running Kubernetes clusters.

Reply:
Ok, I will add to documentation that `init-ed-stack` uses `kubectl`